src/llvm/mod.rs

```diff
@@ -220,7 +220,7 @@ pub unsafe fn optimize(
     }
 
     debug!("running function passes");
-    LLVMInitializeFunctionPassManager(fpm);
+    let _modified = LLVMInitializeFunctionPassManager(fpm) != 0;
```

Comment:
what is the advantage of doing this? the compiler isn't complaining about it, we never check it, the code seems worse to read to me after the change